src/SourceBuild/content/eng/build.sourcebuild.targets

```diff
@@ -1,7 +1,9 @@
 <Project>
 
   <PropertyGroup>
-    <SmokeTestsDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'test', 'Microsoft.DotNet.SourceBuild.SmokeTests'))</SmokeTestsDir>
+    <SmokeTestProjectName>Microsoft.DotNet.SourceBuild.SmokeTests</SmokeTestProjectName>
+    <SmokeTestsDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'test', '$(SmokeTestProjectName)'))</SmokeTestsDir>
```

Comment:
Now that we will add more tests under `~/tests` it would make sense to add a property to the root Directory.Build.props for the tests directory.

Reply:
This is made obsolete by https://github.com/dotnet/installer/pull/19290